### PR TITLE
Fix #161

### LINKED ISSUE #161: Shorten runProcess[Async]/runDartScript
In the spirit of `log`, `fail`, `copy`, `delete`, might be nice to shorten up `runProcess`, `runProcessAsync`, and `runDartScript`.  `run` / `runAsync` seems clear enough to me without `Process`:

`run('dart', ['--version'])`

For `runDartScript` we could crib off of e.g. `Pub.run`, `Dart2js.compile`, etc. and have a `Dart.run`.

`Dart.run('tool/foo.dart')`

 @devoncarew

### PR DESCRIPTION
**Pull request by seaneagan**
@devoncarew

### CODE REVIEW

File: lib/src/run.dart [hunk 1]
```diff
@@ -0,0 +1,143 @@
+
```
Comment (devoncarew) [inline lib/src/run.dart]: Can you add a copyright header?

File: lib/src/run_utils.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (devoncarew) [inline lib/src/run_utils.dart]: copyright header

File: lib/grinder_tools.dart [hunk 3]
```diff
@@ -16,10 +15,14 @@ import 'package:cli_util/cli_util.dart' as cli_util;
 import 'package:which/which.dart';
 
 import 'grinder.dart';
+import 'src/run.dart' as r;
```
Comment (devoncarew) [inline lib/grinder_tools.dart]: Hmm, is there a bit better import prefix for this? `r` seems brief...

File: lib/src/run_utils.dart (hunk 2 above)
Comment (seaneagan) [inline lib/src/run_utils.dart]: done

File: lib/src/run.dart (hunk 1 above)
Comment (seaneagan) [inline lib/src/run.dart]: done

File: lib/grinder_tools.dart (hunk 3 above)
Comment (seaneagan) [inline lib/grinder_tools.dart]: Changed to `run_lib`.  I originally had `run`, but it conflicted with `Pub.run` etc.

**Comment by devoncarew** [comment]
lgtm!

**Pull request merged**